Customize the default Makefile.Linux to use the appropriate flags
Author: Davide Vanzo (Vanderbilt University)
diff -ru S4-7fd00a231610bff51f5c7de5f723e3956eab7453.orig/Makefile.Linux S4-7fd00a231610bff51f5c7de5f723e3956eab7453/Makefile.Linux
--- S4-7fd00a231610bff51f5c7de5f723e3956eab7453.orig/Makefile.Linux	2018-06-10 00:01:13.000000000 -0500
+++ S4-7fd00a231610bff51f5c7de5f723e3956eab7453/Makefile.Linux	2019-06-13 12:29:56.492188945 -0500
@@ -1,17 +1,17 @@
 OBJDIR = build
 SHLIB_EXT = so
 SHLIB_FLAGS = -fPIC -shared
-LA_LIBS = -llapack -lblas
-LUA_INC = -I/usr/include/lua5.2
-LUA_LIB = -llua5.2
+LA_LIBS = -lopenblas
+LUA_INC := -I${EBROOTLUA}/include
+LUA_LIB = -llua -L${EBROOTLUA}/lib
 LUA_MODULE_LIB =
 
-MPI_INC =
-MPI_LIB =
+MPI_INC := ${EBROOTOPENMPI}/include 
+MPI_LIB := ${EBROOTOPENMPI}/lib 
 
-CFLAGS   += -O3 -Wall -march=native -fcx-limited-range -fno-exceptions -fPIC
-CXXFLAGS += -O3 -Wall -march=native -fcx-limited-range -fno-exceptions -fPIC
+CFLAGS   := -Wall -fcx-limited-range -fno-exceptions -ldl ${CFLAGS}
+CXXFLAGS := -Wall -fcx-limited-range -fno-exceptions -ldl ${CXXFLAGS}
 
 # If compiling with MPI, the following must be modified to the proper MPI compilers
-CC = gcc
-CXX = g++
+CC := ${CC}
+CXX := ${CXX}
